flags.go

```diff
@@ -201,6 +334,12 @@ func (g *goforit) SetStalenessThreshold(threshold time.Duration) {
 	g.stalenessThreshold = threshold
 }
 
+func (g *goforit) AddDefaultTags(tags map[string]string) {
+	for k, v := range tags {
```

Comment:
Protect with a mutex?

Reply:
yes, was meaning to add this, just did! thanks!